packages/commonwealth/client/scripts/views/Sublayout.tsx

```diff
@@ -13,7 +13,6 @@ import { SublayoutBanners } from './SublayoutBanners';
 import { SublayoutHeader } from './SublayoutHeader';
 import { AdminOnboardingSlider } from './components/AdminOnboardingSlider';
 import { Breadcrumbs } from './components/Breadcrumbs';
-import GatingGrowl from './components/GatingGrowl/GatingGrowl';
```

Comment:
lets delete this component and all its zustand state code as well

Reply:
All set

Reply:
This is a great catch @mzparacha. I almost wrote in the ticket to make sure we're cleaning up any no longer needed code (incl zustand)